src/core/Entry.cpp

```diff
@@ -711,8 +734,10 @@ QString Entry::resolvePlaceholderRecursive(const QString &placeholder, int maxDe
     case PlaceholderType::NotPlaceholder:
         return placeholder;
     case PlaceholderType::Unknown:
-        qWarning("Can't resolve placeholder %s for entry with uuid %s", qPrintable(placeholder),
-                 qPrintable(uuid().toHex()));
+#ifdef QT_DEBUG
```

Comment:
Just another thing, can you remove this warning? Since we will provide nightly build (see #1197) we discussed on IRC that is better to not have this warning at all.
 
Users will see themselves that the fail reference don't work